docs/reference/cache.md

```diff
@@ -1,36 +1,56 @@
 # Caching in Ratify
 
-Ratify supports many levels of in-memory and file-store-based caching in order to reduce operation overhead and lower the overall verification and mutation duration.
+Ratify supports memory caches and file-based blob caching. 
```

Comment:
Suggestion to add link to "ratify-high-availability.md" so customers reading about this easily find the setup guide.

Reply:
I see there is a link below, now sure if the link should sit at high level so its more discoverable?

Reply:
added link